modules/log-upgrade/6x_creating-and-configuring-a-service-account-for-the-log-collector.adoc

```diff
@@ -0,0 +1,59 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-05-20
+:_mod-docs-content-type: PROCEDURE
+
+[id="creating-and-configuring-a-service-account-for-the-log-collector_{context}"]
+= Creating and configuring a service account for the log collector
+
+Create a service account for the log collector and assign it the necessary roles and permissions to collect logs.
+
+.Prerequisites
+
+.Procedure
+
+. Create a service account to be used by the log collector:
++
+[source,terminal]
+----
+$ oc create sa logging-collector -n openshift-logging
+----
+
+. Bind the `Cluster Role` role to the service account to be able to write the logs to the Red{nbsp}Hat LokiStack
```

Comment:
Cluster Role should be probably all together like "ClusterRole", the same for "service account" as serviceAccount.

Then, probably, it should be like "Bind the `ClusterRole`  to the `serviceAccount` to be able to write the logs

Reply:
done